### PR TITLE
Add next button to verification page

### PR DESCRIPTION
**Pull request by yyorkchan**
Now with javascript enable otp looks like this:
Screenshot 2024-07-09 at 3 03 45 PM

When javascript disabled otp looks like this:
Screenshot 2024-07-09 at 3 05 33 PM

Note that when javascript disabled the login page logo default position is on the left:
Screenshot 2024-07-09 at 3 05 56 PM

### CODE REVIEW

File: resources/authgear/templates/en/web/authflowv2/__otp_input.html [hunk 1]
```diff
@@ -88,5 +81,16 @@
...
+
+  <button
+    form="{{ $.FormName }}"
+    class="primary-btn w-full"
+    type="submit"
+    name="{{ or $.SubmitButtonName "x_action" }}"
+    value="{{ or $.SubmitButtonValue "submit" }}"
+    data-otp-input-target="submit"
+    data-authgear-event="{{ $.SubmitEvent }}"
+  > Next </button>
```
Comment (cychiuae) [inline resources/authgear/templates/en/web/authflowv2/__otp_input.html:93]: Label need to be localised

File: authui/src/authflowv2/otpInput.ts [hunk 2]
```diff
@@ -26,10 +26,13 @@ export class OtpInputController extends Controller {
   };
 
   connect(): void {
+    this.inputTarget.classList.remove("input");
+    this.inputTarget.classList.add("shadow");
     this.inputTarget.addEventListener("input", this.handleInput);
     this.inputTarget.addEventListener("paste", this.handlePaste);
     this.inputTarget.addEventListener("focus", this.handleFocus);
     this.inputTarget.addEventListener("blur", this.handleBlur);
+    this.submitTarget.disabled = true;
```
Comment (louischan-oursky) [inline authui/src/authflowv2/otpInput.ts:35]: You revert the things you do in connect() in disconnect()

```
this.inputTarget.classList.add("input");
this.inputTarget.classList.remove("with-js");
this.submitTarget.disabled = false;
```

File: authui/src/authflowv2/components/otp-input.css [hunk 3]
```diff
@@ -53,17 +51,18 @@
...
       @apply w-full;
-      @apply h-full;
-      @apply outline-none;
-      @apply selection:bg-white/0;
-      @apply caret-white/0;
-      /* iOS treats element with `opacity: 0` as not interactable, set text to transparent instead */
-      @apply text-transparent;
-      background-color: var(--otp-input__bg-color);
+
+      &.shadow {
```
Comment (louischan-oursky) [inline authui/src/authflowv2/components/otp-input.css:57]: &.with-js

File: authui/src/authflowv2/otpInput.ts [hunk 4]
```diff
@@ -26,10 +26,13 @@ export class OtpInputController extends Controller {
   };
 
   connect(): void {
+    this.inputTarget.classList.remove("input");
+    this.inputTarget.classList.add("shadow");
```
Comment (louischan-oursky) [inline authui/src/authflowv2/otpInput.ts:30]: with-js

File: resources/authgear/templates/en/web/authflowv2/__otp_input.html (hunk 1 above)
Comment (louischan-oursky) [reply resources/authgear/templates/en/web/authflowv2/__otp_input.html:93]: {{ template "v2-button-label-continue" }}

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**